packages/rule/stories/rule.stories.ts

```diff
@@ -0,0 +1,236 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import { html, TemplateResult } from 'lit-html';
+
+import '../';
+import '@spectrum-web-components/button';
+import '@spectrum-web-components/icon';
+import {
+    AlignLeftIcon,
+    AlignRightIcon,
+} from '@spectrum-web-components/icons-workflow';
+
+export default {
+    title: 'Rule',
+    component: 'sp-rule',
+    decorators: [
+        (story): TemplateResult => html`
+            <style>
+                sp-theme {
+                    color: var(--spectrum-global-color-gray-800);
+                }
+                .spectrum-Heading--subtitle1 {
```

Comment:
Kind of annoying to have to recreate these here.

Reply:
Yeah, it has to do with needing them out of the `node_modules` scope.

I could try again to include them in Storybook's `preview.js`, but I'm pretty sure that while using the version from Open WC we don't get magic compiling there and it'll choke.

Another option is to build a demo element that has `@spectrum-web-components/styles/typography.css` in it's styles and accepts a TemplateResult, a la:
```
<typography-demo
    .content=${html`
        <h2 class="spectrum-Heading--subtitle1">Large</h2>
        <sp-rule size="large"></sp-rule>
        <p class="spectrum-Body">Page or Section Titles.</p>
    `}
></typography-demo>
```
Could keep that in the decorator, and allow the stories to go unchanged... 🤔 That's more interesting then when I thought it through before. I'll try again.

Reply:
This pattern is definitely cleaner, but it might start to open a separate can of worms around having more of the content in `@spectrum-web-components/styles` available as `CSSResultsArrays`, which is a sledgehammer of a solution to styling that it probably best we don't actively support. I don't have enough use cases to fell conclusively on this, however, as we aren't currently using a lot of typography or styles beyond the components themselves in our project.

@cuberoot Does this seem more like what you were expecting here?

Reply:
Yah, that seems better. At least we don't have to cut-and-paste to use the typography to use it in our stories. We can think about how we would want to expose typography in future...